test/functional/ui/searchhl_spec.lua

```diff
@@ -25,20 +25,6 @@ describe('search highlighting', function()
     screen:set_default_attr_ignore( {{bold=true, foreground=hl_colors.NonText}} )
   end)
 
-  it('is off by default', function()
-    insert("some text\nmore text")
-    feed("gg/text<cr>")
-    screen:expect([[
-      some ^text                               |
-      more text                               |
-      ~                                       |
-      ~                                       |
-      ~                                       |
-      ~                                       |
-      /text                                   |
-    ]])
-  end)
```

Comment:
It seems that `set hlsearch` is not tested below, so why not add `execute('set hlsearch')` to line 29 and keep this spec?